receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -123,14 +125,15 @@ func (prw *prometheusRemoteWriteReceiver) handlePRW(w http.ResponseWriter, req *
 		return
 	}
 
-	_, stats, err := prw.translateV2(req.Context(), &prw2Req)
+	m, stats, err := prw.translateV2(req.Context(), &prw2Req)
 	stats.SetHeaders(w)
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest) // Following instructions at https://prometheus.io/docs/specs/remote_write_spec_2_0/#invalid-samples
 		return
 	}
 
 	w.WriteHeader(http.StatusNoContent)
+	_ = prw.nextConsumer.ConsumeMetrics(req.Context(), m)
```

Comment:
Should I deal with that error?

Reply:
typically these errors are returned to the obsreport object capturing the ingestion. The error is also logged.

Reply:
[2451c27](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38812/commits/2451c2780651669016526d27a49337baf76da82d)

Reply:
I added a TODO comment here to address it in the future, this because this change is unrelated to the target_info feature